src/main/groovy/com/mini/asaas/utils/message/MessageType.groovy

```diff
@@ -0,0 +1,12 @@
+package com.mini.asaas.utils.message
+
+enum MessageType {
+
+    SUCCESS,
+    ERROR,
+    WARNING
+
+    String toLowerCase() {
+        return this.name().toLowerCase()
+    }
```

Comment:
Esse método é usado em algum lugar? Acabei não encontrando aqui.

Se não for, pode remoevr. 
Se for utilizar em um PR futuro, é preferível criar no mesmo PR em que é utilizado, pois fica evidente pra que vai ser usado